core/Cargo.toml

```diff
@@ -30,6 +29,7 @@ rw-stream-sink = { workspace = true }
 serde = { version = "1", optional = true, features = ["derive"] }
 smallvec = "1.11.0"
 thiserror = "1.0"
+tracing = { version = "0.1.37", features = ["log"] }
```

Comment:
I think activating `log` is something that downstream users should do!

Reply:
got it, ill update relevant tracing imports to remove the `log` feature